src/SFA.DAS.AANHub.Domain/Interfaces/IDateTimeProvider.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.AANHub.Domain.Interfaces
+{
+    public interface IDateTimeProvider
+    {
+        DateTime Now() => DateTime.UtcNow;
```

Comment:
This looks wrong.  Shouldn't this be something like 
public DateTime Now {get;}